app/js/actions/fetchOrders.js

```diff
@@ -9,7 +9,7 @@ const fetchOrders = (
 ) => ({
   type: FETCH_ORDERS,
   payload: axiosInstance
-    .get(uri || `/order?totalCount=true&sort=desc&status=active&patient=${patient}&limit=${limit}&v=full`),
+    .get(uri ? uri.slice(51) : `/order?totalCount=true&sort=desc&status=active&patient=${patient}&limit=${limit}&v=full`),
```

Comment:
Why 51 and not 52?

Reply:
I'm using 51 because `/` is needed. 
If I had used 52, I would have gotten a URL like `order?totalCount=true&sort=desc&status=active&patient=82af813c-5118-4ae9-938c-927284453957&limit=10&v=full&startIndex=20` instead of `/order?totalCount=true&sort=desc&status=active&patient=82af813c-5118-4ae9-938c-927284453957&limit=10&v=full&startIndex=20`

Reply:
What constitutes the 51 characters?

Reply:
`http://localhost:8081/openmrs-standalone/ws/rest/v1`

Reply:
Will the same code work for? http://localhost:8081/openmrs/ws/rest/v1

Or https://demo.openmrs.org/openmrs/ws/rest/v1

Reply:
No, it will not. I will fix that. Thanks.

Reply:
It has been fixed.